camelot/admin/menu.py

```diff
@@ -50,6 +50,7 @@ class MenuItem(DataclassSerializable):
     action_route: typing.Union[Route, None] = None
     role: typing.Optional[str] = None
     items: typing.List['MenuItem'] = field(default_factory=list)
+    open: bool = False
```

Comment:
Dit heb ik toegevoegd, maar dit is waarschijnlijk niet nodig vermits we die open/highlighted enkel expliciet toevoegen voor de unit tests?